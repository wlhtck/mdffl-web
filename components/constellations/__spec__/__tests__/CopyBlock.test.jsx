import React from 'react';
import { render } from 'enzyme';
import ContentBlockCopy from '../../CopyBlock';

describe('components/constellations/ContentBlock/Copy', () => {
  it('renders <ContentBlockCopy /> with a CTA', () => {
    expect(render(
      <ContentBlockCopy
        headline="About Us"
        body="We are a LGBT friendly league. Itâ€™s a very competitive league so bring your A-game."
        cta={{
          href: '/about',
          children: 'LEARN MORE',
          external: false,
        }}
      />,
    )).toMatchSnapshot();
  });
  it('renders <ContentBlockCopy /> without a CTA', () => {
    expect(render(
      <ContentBlockCopy
        headline="Mission Statement"
        textAlign="left"
        body="
        The Metro Detroit Flag Football League (MDFFL) is an LGBT and straight allied flag football
        league, changing preconceived societal beliefs about sexual orientation and
        self-identification.
        "
      />,
    )).toMatchSnapshot();
  });
});
