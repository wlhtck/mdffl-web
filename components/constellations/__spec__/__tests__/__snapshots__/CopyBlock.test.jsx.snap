// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/constellations/ContentBlock/Copy renders <ContentBlockCopy /> with a CTA 1`] = `
<div
  class="withStyles-u5urts-0 efexQC"
>
  <h2
    class="withStyles-u5urts-0 jLXugA"
  >
    About Us
  </h2>
  <div
    class="withStyles-u5urts-0 cjEcvK"
  >
    <p>
      We are a LGBT friendly league. Itâ€™s a very competitive league so bring your A-game.
    </p>
  </div>
  <a
    class="withStyles-u5urts-0 gqQEcA"
    href="/about"
    type="primary"
  >
    LEARN MORE
  </a>
</div>
`;

exports[`components/constellations/ContentBlock/Copy renders <ContentBlockCopy /> without a CTA 1`] = `
<div
  class="withStyles-u5urts-0 efexQC"
>
  <h2
    class="withStyles-u5urts-0 jLXugA"
  >
    Mission Statement
  </h2>
  <div
    class="withStyles-u5urts-0 cjEcvK"
  >
    <p>
       The Metro Detroit Flag Football League (MDFFL) is an LGBT and straight allied flag football league, changing preconceived societal beliefs about sexual orientation and self-identification. 
    </p>
  </div>
</div>
`;
